library/src/main/java/org/xmtp/android/library/Conversations.kt

```diff
@@ -318,23 +301,23 @@ data class Conversations(
         return Dm(client, dmConversation)
     }
 
-    suspend fun newConversationWithInboxId(
-        peerInboxId: String,
+    suspend fun newConversation(
+        peerInboxId: InboxId,
```

Comment:
Big breaking change here is newConversation and new Group now takes a inboxId instead of a address... but because they are both strings it will blow up only after it's been hit in the code.